src/OneExplorer/OneExplorer.ts

```diff
@@ -229,6 +229,39 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
         });
   }
 
+  delete(oneNode: OneNode): void {
+    const isDirectory = (oneNode.node.type === NodeType.directory);
+    const title =
+        `Are you sure you want to delete '${path.parse(oneNode.node.path).base}'` + isDirectory ?
+        'and its contents?' :
+        '?';
+    const detail = `You can restore this file from the Trash.`;
+    const approval = 'Move to Trash';
+    const recursive = isDirectory ? true : false;
+
+    if (oneNode.node.type === NodeType.directory) {
+      vscode.window.showInformationMessage(title, {detail: detail, modal: true}, approval)
+          .then(ans => {
+            if (ans === approval) {
+              Logger.info('OneExplorer', `Delete '${oneNode.node.name}'.`);
+              return vscode.workspace.fs.delete(
+                  oneNode.node.uri, {recursive: recursive, useTrash: true});
+            }
+          })
+          .then(() => this.refresh());
```

Comment:
This is identical to L253 to L261.

how about changing the current code

```
if (oneNode.node.type === NodeType.directory) {
  vscode.window.showInformationMessage(...);
} else {
  vscode.window.showInformationMessage(...);  // same with the above showInformationMessage
}
```

to  
```
vscode.window.showInformationMessage(...);
```

Reply:
Cool, I missed it. Thanks :-D